src/lib/token_list_gen.ts

```diff
@@ -111,11 +112,14 @@ export const generateTokenList = async (
 
   console.log(`List has ${l2TokenList.length} bridged tokens`);
 
+  let unbridgedL1Tokens:ArbTokenInfo[]  = []
   if(options && options.includeUnbridgedL1Tokens){
     const l1AddressesOfBridgedTokens = new Set(tokens.map((token)=> token.l1TokenAddr.toLowerCase()))
-    const unbridgedL1Tokens:ArbTokenInfo[] = l1TokenList.tokens.filter((l1TokenInfo)=>{
+    unbridgedL1Tokens = l1TokenList.tokens.filter((l1TokenInfo)=>{
       return !l1AddressesOfBridgedTokens.has(l1TokenInfo.address.toLowerCase()) && l1TokenInfo.chainId === +l1Network.chainID
-    }).map((l1TokenInfo)=>{
+    }).filter(
+      (l1TokenInfo) => l1TokenInfo.chainId !== parseInt(l2Network.chainID)
```

Comment:
Don't we want to check the chainId is actually equal to the `l1Network.chainID`?
failure mode here would be if a list contains tokens from both rinkeby and ropsten L1 networks
```suggestion
      (l1TokenInfo) => l1TokenInfo.chainId === parseInt(l1Network.chainID)
```

Reply:
Some lists (like uniswap) already contain token addresses for multiple chains, so I didnt want to filter the other chains out. My thinking was that when we arbify a list we download a list of l1 tokens and generate new l2 tokens. So when we arbify the list we want to append everything that's not the l2 token - then generate new l2 tokens and append those.

Reply:
This brings up the annoying point that's there's more than on "arbified" version of a given list — i.e., the "Arbitrum One Mainnet - arbified list" vs. the "RinkArby Arbified list." I suppose ideally the script would check and arbify for all supported Arbitrum chains (I don't think we need to do that in this PR tho)